### DISCUSSION TITLE
Export process job stopped working some versions back - not sure which ?

### DISCUSSION THREAD

**Discussion by vw-kombi** [arabcoders/watchstate#487] [General] [answered]
I run on unraid.
I have the import that imports from my live emby server as part of a backup routine, then export to the backjup emby server.
The import is fine, but the export is not.
I also tried the same on the GUI console command - same error.
Do I need to re-create the backend or something ?

This is the command :
  docker exec WatchState console state:export -v -s embybackup 

This is the output - 

[2024-06-11T23:14:48+00:00] NOTICE: DATABASE: Loading changed items since [2024-05-25 06:02:16 UTC].

In helpers.php line 86:

[TypeError]
makeDate(): Argument #1 ($date) must be of type DateTimeInterface|string|in
t, null given, called in /opt/app/src/Commands/State/ExportCommand.php on l
ine 349

Exception trace:
at /opt/app/src/Libs/helpers.php:86
makeDate() at /opt/app/src/Commands/State/ExportCommand.php:349
App\Commands\State\ExportCommand->process() at /opt/app/src/Commands/State/ExportCommand.php:134
App\Commands\State\ExportCommand->App\Commands\State\{closure}() at /opt/app/src/Command.php:148
App\Command->single() at /opt/app/src/Commands/State/ExportCommand.php:134
App\Commands\State\ExportCommand->runCommand() at /opt/app/src/Command.php:75
App\Command->execute() at /opt/app/vendor/symfony/console/Command/Command.php:326
Symfony\Component\Console\Command\Command->run() at /opt/app/vendor/symfony/console/Application.php:1078
Symfony\Component\Console\Application->doRunCommand() at /opt/app/vendor/symfony/console/Application.php:324
Symfony\Component\Console\Application->doRun() at /opt/app/vendor/symfony/console/Application.php:175
Symfony\Component\Console\Application->run() at /opt/app/src/Libs/Initializer.php:174
App\Libs\Initializer->console() at /opt/app/bin/console:69

state:export [-f|--force-full] [--dry-run] [--timeout TIMEOUT] [-s|--select-backend [SELECT-BACKEND]] [--exclude] [-i|--ignore-date] [--logfile LOGFILE]

**Comment by arabcoders** [comment, marked as answer]
Thanks for reporting the issue. it should be fixed soon, The bug is in a code path that rarely used which is why it was probably never fixed. 
The relevant issue is #489
